src/index.html

```diff
@@ -7,6 +7,44 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="
+        calendar__field
+        calendar--sat
+        calendar--32
+        ">
```

Comment:
```suggestion
      <div class="calendar__field calendar--sat calendar--32">
```
В одну строку вроде тоже влезает